project/frontend/src/components/ResultsMap.js

```diff
@@ -7,27 +7,36 @@ function MapContainer(props) {
   const userCenter = props.userLocation;
   const [activeMarker, setActiveMarker] = useState({});
   const [showInfoWindows, setShowInfoWindows] = useState({
+    marker0: false,
     marker1: false,
     marker2: false,
     marker3: false,
-    marker4: false,
   });
 
   // TODO: Add marker for "you are here location".
   const MarkerIcon = (props) => {
-    const markerName = 'marker' + props.id;
+    const markerID = props.id;
+    const markerName = 'marker' + markerID;
```

Comment:
nit: markerID seems pretty self explanatory - you could maybe just simplify this to `const makerName = `marker ${props.id}``

Reply:
the tick marks there need to be back ticks `

Reply:
But it's a handy trick when creating chained strings with variables so you do have to manually add + and extra spaces etc
read more about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
this is good to know!! thank you :)